src/components/Iconography/Iconography.stories.mdx

```diff
@@ -0,0 +1,176 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import { getCategory } from "../../utils/componentCategories";
+import Icon from "../Icons/Icon";
+import { IconColors, IconSizes } from "../Icons/IconTypes";
+
+<Meta
+  title={getCategory("Iconography")}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10975%3A16",
+    },
+  }}
+/>
+
+# Iconography
+
+Use the NYPL Design System [Icon](https://nypl.github.io/nypl-design-system/storybook-static/?path=/docs/icon--control-props) component to render and control icons and select NYPL logos as `<svg>` elements within consuming apps. Below are a few example icons.
```

Comment:
Can you add that you can select an icon name by using the `IconNames` or `LogoNames` enums? I would have imagined that the `name="check"` prop would throw an error...

Reply:
I added additional verbiage about the enum values.